docs/validators/redundancy.md

```diff
@@ -0,0 +1,51 @@
+---
+id: redundancy
+title: Node redundancy
+---
+
+Elrond Validator Nodes can be configured to have one or more hot-standby nodes. 
+This means additional nodes will run on different servers, in sync with the Main Validator node. 
+Their role is to stand in for the Main Validator node in case it fails, to ensure high availability.
+
+This is a redundancy mechanism which allows the Main Validator operator to start additional 'n' hot-standby nodes, 
+each of them running the same 'validatorKey.pem' file. The difference between 
+configurations consists on an option inside the `prefs.toml` file.
+
+Hot standby nodes are configured using the 'RedundancyLevel' option in the 'prefs.toml' configuration file:
+
+- a 0 value will represent that the node is the Main Validator.
+The value 0 will be the default, if the option is missing (as we do not re-write the `prefs.toml` 
+  files during the node's upgrade) will still make that node the Main Validator: 
+  backwards compatibility holds.
```

Comment:
```suggestion
The value 0 will be the default, therefore if the option is missing it will still make that node the Main Validator by default. With consideration to backwards compatibility, the already-running Validators are not affected by the addition of this option. Moreover, we never overwrite the `prefs.toml` files during the node's upgrade.

The values of `RedundancyLevel` are interpreted as follows:
```